Washington-County/COUNTY-DRUPAL
# WCOR-1050: Identify and add language set for core multilingual.

**ODE 95**

**Use case**

As an editor, I want to be able to enter translated website content in the County's safe harbor languages.

**Implementation details**

1. Confirm the County's safe harbor languages with Philip. (Update: Philip confirmed the list below.)

   - Arabic
   - Chinese (Cantonese & Mandarin)
   - Filipino/Tagalog
   - Japanese
   - Khmer/Cambodian
   - Korean
   - Persian
   - Russian
   - Somali
   - Spanish
   - Vietnamese

2. Add language names and language codes in Drupal via /admin/config/regional/language.

**Test plan**

1. Create a new page in Drupal.
2. Click the "Translate" tab on the node once it's saved.
3. Verify that English and all the County's safe harbor languages are listed.
4. Click "Add" to the far right of any language to enter a translation of the English page you created.
5. Replace English content with sample translated content in the required fields.
6. Click "Save (all translations)."
7. Verify that the new translated page gets a path beginning with the language code. For example, if the English URL is /administration-hhs-directors-office/test, the Spanish URL would begin with /es/ and might be something like /es/administration-hhs-directors-office/prueba.
